docker/images/humble/Dockerfile

```diff
@@ -91,7 +91,7 @@ COPY --from=cacher --chown=$USER:$GROUP /tmp/ws/ $USER_WS/
 
 RUN sudo apt-get update \
   && rosdep update \
-  && rosdep install -i --from-path src -y \
+  && ROS_VERSION=2 rosdep install -i --from-path src -y \
```

Comment:
Hmm, I should source the distro instead, that should set `ROS_VERSION` in the environment.

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/218/commits/6dc705c6da6cae1fb20143885b5654b9acb48ed4.